src/main/resources/swagger/rawls.yaml

```diff
@@ -227,6 +227,167 @@ paths:
             - openid
             - email
             - profile
+  '/admin/groups':
+      post:
+        responses:
+          '201':
+            description: Created
+          '403':
+            description: Unauthorized
+            schema:
+              $ref: '#/definitions/ErrorReport'
+          '500':
+            description: Rawls Internal Error
+            schema:
+              $ref: '#/definitions/ErrorReport'
+        description: Create a group
+        tags:
+          - admin
+        summary: create a group
+        operationId: adminCreateGroup
+        parameters:
+          - in: body
+            description: RawlsGroupRef
+            name: group
+            required: true
+            schema:
+              $ref: '#/definitions/RawlsGroupRef'
+        security:
+          - authorization:
+              - openid
+              - email
+              - profile
+
+  '/admin/groups/{groupName}':
+      delete:
+        responses:
+          '204':
+            description: Successful Request
+          '403':
+            description: Unauthorized
+            schema:
+              $ref: '#/definitions/ErrorReport'
+          '500':
+            description: Rawls Internal Error
+            schema:
+              $ref: '#/definitions/ErrorReport'
+        description: Delete a group
+        tags:
+          - admin
+        summary: delete a group
+        operationId: adminDeleteGroup
+        parameters:
+          - in: path
+            description: name of the group
+            name: groupName
+            required: true
+            type: string
+        security:
+          - authorization:
+              - openid
+              - email
+              - profile
+
+  '/admin/groups/{groupName}/members':
+      get:
+        responses:
+          '201':
+            description: Successful Request
+            schema:
+              type: array
+              items:
+                type: string
```

Comment:
Do we want to distinguish the users from the groups when listing group members in some structured way?